tensorflow/python/ops/math_ops.py

```diff
@@ -3298,7 +3298,7 @@ def log_sigmoid(x, name=None):
   
   For example:
   >>> tf.math.log_sigmoid(tf.constant(1.0, tf.float32))
-  <tf.Tensor: shape=(), dtype=float32, numpy=-0.31326166>
+  <tf.Tensor: shape=(), dtype=float32, numpy=-0.31>
```

Comment:
If you don't add the ... after 0.31 the doctest will fail. Ellipsis are interpreted as wildcard matches.

Reply:
Hello, I had made the changes requested. Can you please take a look?